python/llm/src/bigdl/llm/serving/README.md

```diff
@@ -48,10 +48,10 @@ python3 -m fastchat.serve.controller
 
 ### Launch model worker(s) and load models
 
-#### BigDL model worker
-
 Using BigDL-LLM in FastChat does not impose any new limitations on model usage. Therefore, all Hugging Face Transformer models can be utilized in FastChat.
 
+
+#### BigDL model worker (deprecated...)
```

Comment:
can we fold this part or make the fonts grey
add a notice to user for new BigDL-LLM worker

Reply:
Let me try this

Reply:
This part has been folded.